modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -4708,7 +4708,7 @@ export const console: ConsoleNS = {
                             description: "View and manage how user attributes are mapped and " +
                                 "transformed when interacting with APIs or your applications.",
                             primaryAction: "New Attribute Mapping",
-                            title: "Attributes and Mappings",
+                            title: "Attributes",
```

Comment:
Shouldn't we do the same in the side panel as well? @yasinmiran

Reply:
cc:/ @ayshsandu

Reply:
ACK. Got the [confirmation](https://github.com/wso2-enterprise/asgardeo-product/issues/2725#issuecomment-803845089) and applied the changes via commit e933a0f